composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4",
+        "php": "^8.0.2",
         "ext-pdo": "*",
         "mautic/core-lib": "^3.0|^4.0"
```

Comment:
Probably mautic/core-lib:^3.0 can be removed here as it does not run on PHP 8

Reply:
Resolved with commit "Update composer.json"